### PR TITLE
i3ipc 0.3

### PR DESCRIPTION
**Pull request by Armael**
cc @chimrod

### CODE REVIEW

File: packages/i3ipc/i3ipc.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+dev-repo: "git+https://github.com/Armael/ocaml-i3ipc"
+doc: "https://armael.github.io/ocaml-i3ipc/0.3/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "lwt"
+  "lwt_ppx"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/i3ipc/i3ipc.0.3/opam]:
```suggestion
  "dune" {>= "1.8"}
```
The `dune-project` file in your sources says this package requires at least dune 1.8

Comment (Armael) [reply packages/i3ipc/i3ipc.0.3/opam]: Good catch. I've instead relaxed the constraint on the version in the dune-project; it now should work for dune >= 1.0, ie any version of dune.

**Comment by Chimrod** [comment]
Merci !

I'll update my code with your new protocol_error printers as soon as your version will be publish on opam

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**